six/common/builtins/aggregator.c

```diff
@@ -95,7 +94,7 @@ static char **py_tag_to_c(PyObject *py_tags)
         return tags;
     }
 
-    py_tags_list = PySequence_Fast(py_tags, err); // new reference
+    py_tags_list = PySequence_Fast(py_tags, "py_tags is not a sequence"); // new reference
```

Comment:
We should maybe start adding/consolidating error message strings, into header(s) for consistency (as a best practice). It's a bit of a chore and maybe beyond the scope of this PR, we could tackle that separately.